### PR TITLE
refactor(3491): rename animation duration variables and revert base variab…

### PR DESCRIPTION
**Pull request by anveshmekala**
**Related Issue:** #3491 

## Summary

- This will rename the `--calcite-animation-timing-factor` to `--calcite-duration-factor` be more concise and avoids ambiguity of whether it is used for animations only or transitions included.  
- Additionally, reverted the change introduced in #3492 where the base global animation timing variable `--calcite-animation-timing` is moved to  `@internal`. With this PR, `--calcite-animation-timing` will be a global variable
- Rename the  internal animation timing scale `quick` introduced in #3492 to `fast`

### CODE REVIEW

File: src/tests/globalStyles.e2e.ts [hunk 1]
```diff
@@ -38,17 +38,17 @@ describe("global styles", () => {
...
 
-    it("should set animation duration to 0ms when --animation-timing-factor set to zero", async () => {
+    it("should set animation duration to 0ms when --calcite-duration-factor set to zero", async () => {
       const page = await newE2EPage({
         html: html`
           <html>
             <style>
               html {
-                --calcite-animation-timing-factor: 0;
+                --calcite-duration: 0;
```
Comment (jcfranco) [inline src/tests/globalStyles.e2e.ts:47]: This should be `--calcite-duration-factor`, right? This also applies to the next test.

File: src/assets/styles/_animation.scss [hunk 2]
```diff
@@ -41,18 +41,18 @@
 }
 
 :root {
-  --calcite-internal-animation-timing-factor: var(--calcite-animation-timing-factor, 1);
-  --calcite-internal-animation-timing: calc(150ms * var(--calcite-internal-animation-timing-factor));
-  --calcite-internal-animation-timing-quick: calc(100ms * var(--calcite-internal-animation-timing-factor));
-  --calcite-internal-animation-timing-medium: calc(200ms * var(--calcite-internal-animation-timing-factor));
-  --calcite-internal-animation-timing-slow: calc(300ms * var(--calcite-internal-animation-timing-factor));
+  --calcite-internal-duration-factor: var(--calcite-duration-factor, 1);
+  --calcite-animation-timing: calc(150ms * var(--calcite-internal-duration-factor));
```
Comment (jcfranco) [inline src/assets/styles/_animation.scss:45]: Nitpick: can you move this one to be first so it's easier to distinguish from the internal ones?

**Review by jcfranco — APPROVED** [review]
Once the test comments are fixed, this is good to merge.

File: src/assets/styles/_animation.scss [hunk 3]
```diff
@@ -41,18 +41,18 @@
 }
 
 :root {
-  --calcite-internal-animation-timing-factor: var(--calcite-animation-timing-factor, 1);
-  --calcite-internal-animation-timing: calc(150ms * var(--calcite-internal-animation-timing-factor));
-  --calcite-internal-animation-timing-quick: calc(100ms * var(--calcite-internal-animation-timing-factor));
```
Comment (macandcheese) [inline src/assets/styles/_animation.scss:46]: Can we use 'fast' here? Nitpick but we already use that terminology on `alert`.

File: src/tests/globalStyles.e2e.ts (hunk 1 above)
Comment (anveshmekala) [reply src/tests/globalStyles.e2e.ts:47]: Yup.

File: src/assets/styles/_animation.scss (hunk 3 above)
Comment (anveshmekala) [reply src/assets/styles/_animation.scss:46]: change the name to `fast` . Good catch @macandcheese

**Pull request merged**